common/src/main/java/org/apache/uniffle/common/ShufflePartitionedBlock.java

```diff
@@ -76,15 +80,12 @@ public boolean equals(Object o) {
       return false;
     }
     ShufflePartitionedBlock that = (ShufflePartitionedBlock) o;
-    return dataLength == that.dataLength
-        && crc == that.crc
-        && blockId == that.blockId
-        && data.equals(that.data);
```

Comment:
Please pay attention to the changes here. @jerqi

Reply:
If we don't have special reason, we should use the common ways. If you have other special reasons, we should add comment to tell others this is not wrong code.